group_vars/all/release.yml

```diff
@@ -13,10 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## Using this file
-#
-# This file is laid down only once and is used for overriding variables based on the file name.
-# This file will not be overwritten by upgrades.
+# The release tag to use for the repo and venvs
+# This can't be overriden because OSA is using group_vars.
+#openstack_release: "{{ rpc_release }}"
+
+# The release tag to use
+# Note that this is set here so that the openstack_release override in osa_defaults
+# is able to use it.
+rpc_release: "{{ lookup('pipe', 'cd /opt/rpc-openstack && git describe --tags --abbrev=0') }}"
```

Comment:
These files were created for N release. They could change anytime as long as we explain. Also it doesn't mean doc change. Userland is still userland.

I can do it, but I will just do it to end this bikeshed.